src/clr-angular/data/stack-view/_stack-view.clarity.scss

```diff
@@ -90,6 +90,10 @@
       }
     }
 
+    .stack-block-label {
+      outline: none;
```

Comment:
Hmm, I have UX concern over this as I think it would create focus state inconsistencies over elements/components. Once a user starts clicking the tab key to navigate through elements/components, native focus actually disappears but its hover state will kick in as a focus state.

Reply:
@shijir: Here the hover and the focus states are the same. We do this in a components like the sidenav, subnav, vertical nav etc too.

Reply:
I should have added a note here though.

Reply:
Wait, we have a native focus for vertical nav and side nav, don't we?

Reply:
Lets confirm how UX wants this

Reply:
discussed with @mathisscott and @Shijir. this is the behavior for now. we can discuss the focus styles later on as a separate change.